modules/nf-core/custom/tabulartogseacls/main.nf

```diff
@@ -0,0 +1,41 @@
+process CUSTOM_TABULARTOGSEACLS {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda (params.enable_conda ? "conda-forge::coreutils=9.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+        'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(samples)
+
+    output:
+    tuple val(meta), path("*.cls"), emit: cls
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    def separator = samples.getName().endsWith(".csv") ? ',' : '\\t'
```

Comment:
Coud this not be passed via `args`? Or is the tabular input definitely always TSV or CSV for the tool?

Reply:
I like the auto behaviour, but I've added an args override and demonstrated using a test in https://github.com/nf-core/modules/pull/2606/commits/81e8089638e8f8183ba8faf168a4e642021fd4d2